hardware_layer/hardware_arduino/README.md

```diff
@@ -107,23 +78,22 @@ First upload the code on arduino through Arduino IDE.
 ### arduino_node
 Subscribes to topics with PWM data and actuate the thrusters with that duty cycle, and also publishes the data obtained from pressure sensor
 
-__NOTE:__ Pins configurations are specified in the [ArduinoConfig.h](include/ArduinoConfig.h) file.
-
 #### Subscribed Topics
-* **`/thruster/surge1/pwm`** ([std_msgs/Int32])
-* **`/thruster/surge2/pwm`** ([std_msgs/Int32])
-* **`/thruster/heave1/pwm`** ([std_msgs/Int32])
-* **`/thruster/heave2/pwm`** ([std_msgs/Int32])
-* **`/thruster/sway1/pwm`** ([std_msgs/Int32])
-* **`/thruster/sway1/pwm`** ([std_msgs/Int32])
+* **`/ard/east`** ([std_msgs/Int32])
+* **`/ard/west`** ([std_msgs/Int32])
+* **`/ard/northsway`** ([std_msgs/Int32])
+* **`/ard/southsway`** ([std_msgs/Int32])
+* **`/ard/neup`** ([std_msgs/Int32])
+* **`/ard/seup`** ([std_msgs/Int32])
+* **`/ard/nwup`** ([std_msgs/Int32])
+* **`/ard/swup`** ([std_msgs/Int32])
 
 #### Published Topics
-* **`/pressure_sensor/pressure`** ([underwater_sensor_msgs/Pressure]): Pressure sensor data (in Pascals)
+* **`/varun/sensors/pressure_sensor/depth`** ([std_msgs/Int32])
```

Comment:
I would advice against having the prefix `varun/senors`. The topic name is becoming too long. Keep it `sensors` only I suppose?